app/javascript/src/components/Editor.tsx

```diff
@@ -363,6 +372,143 @@ class Editor extends React.Component<EditorProps, EditorState> {
     }
   }
 
+  private onCopy = () => {
+    const { model } = this
+
+    const selectedNodes = model
+      .getSelectedItems()
+      .filter(item => item instanceof DefaultNodeModel) as DefaultNodeModel[]
+    const selectedLinks = this.model
+      .getSelectedItems()
+      .filter(item => item instanceof DefaultLinkModel) as DefaultLinkModel[]
```

Comment:
scope creep feature request: If I shift-click on two linked scenes but don't select the joining link, it would be awesome if the joining link was added to the copiedLinks variable. I imagine another doubly nested loop with some complicated logic might do the trick?

Reply:
Added in the latest commit -- It wasn't quite as flat as just two loops, since the way to get Links from Nodes is to go through Ports, which ups the number of loops from having to loop through the ports first.